.github/workflows/release.yaml

```diff
@@ -0,0 +1,52 @@
+name: Weekly Release
+
+on:
+  schedule:
+    - cron: '0 11 * * 1' # Run every Monday at 7am NYC time
+      branches:
+        - master
+  workflow_dispatch: # allows manual releasing
+      branches:
+        - master
+jobs:
+  build_and_release:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+
+      - name: Set up Bazel
+        uses: actions/setup-java@v3
+        with:
+          distribution: 'adopt'
+          java-version: '21'
+
+      - name: Build with Bazel
+        run: bazel build java/com/google/copybara/copybara_deploy.jar
+
+      - name: Get current date
+        id: date
+        run: echo "::set-output name=date::$(date +%Y%m%d)"
+
+      - name: Create Release
+        uses: actions/create-release@v1
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        with:
+          tag_name: v${{ steps.date.outputs.date }}
+          release_name: Release v${{ steps.date.outputs.date }}
+          body: |
+            Automated weekly test release snapshot from master branch.
+            This is a test release, version compatibility or correctness
+            not guaranteed.
+          draft: true # change this to false once it works
+
+      - name: Upload Release Asset
+        uses: actions/upload-release-asset@v1.0.2
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        with:
+          upload_url: ${{ steps.create_release.outputs.upload_url }}
+          asset_path: bazel-bin/java/com/google/copybara/copybara_deploy.jar
+          asset_name: copybara_deploy.jar
```

Comment:
nit: it would be nice to also publish a checksum of the file. That way users can vendor the checksum when they take the dependency (following https://en.wikipedia.org/wiki/Trust_on_first_use principle)

As you probably know, Bazel will require the checksum to cache this jar file. Users could compute one themselves of course.

Reply:
Done